### ISSUE TITLE
7zip@19.00: hash check failed

### ISSUE THREAD

**Issue by scaratec** [ScoopInstaller/Main#927]
I'm running this command:

```scoop install 7zip```

and get this response:

```
scoop install 7zip
Installing '7zip' (19.00) [64bit]
7z1900-x64.msi (2.7 KB) [================================================================================================================================================================================] 100%
Checking hash of 7z1900-x64.msi ... ERROR Hash check failed!
App:         main/7zip
URL:         https://7-zip.org/a/7z1900-x64.msi
First bytes: 3C 68 74 6D 6C 3E 0D 0A
Expected:    a7803233eedb6a4b59b3024ccf9292a6fffb94507dc998aa67c5b745d197a5dc
Actual:      dc710692581b2d509fb1004508825adec78b19a0f096d5c64650a2052f0eaf60

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=7zip%4019.00%3a+hash+check+failed
```

The suggested retry didn't help.

**Comment by Ash258** [comment]
Download size of 7zip installer is about 1.7MB.

Most likely you downloaded webpage which is saying that this domain is blocked by your Firewall/Corporate restrictions/...

**Comment by 0x326** [comment]
Related: #899

**Comment by Ash258** [comment]
https://github.com/ScoopInstaller/Main/issues/899#issuecomment-607390881
https://github.com/ScoopInstaller/Main/issues/927#issuecomment-603835705
